docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -1,11 +1,60 @@
 ## LittleFileSystem
```

Comment:
The use of LittleFS on this page is overwhelming and confusing. The first paragraph is arguable but the rest is absurd.

Reply:
How so?

The name is defined here:
> The little filesystem (LittleFS) is a fail-safe filesystem designed for embedded systems, specifically for microcontrollers that use flash storage.

At some point the following docs need to refer to the name of the filesystem to identify what it's talking about. Back when we had LWIPInterface, the docs all refered to the stack as LWIP did it not?

Reply:
I'd hope not and if so should also be changed. You're on a page titled docs/ref/ver/api/storage/LittleFilesystem That seems pretty clear to me :)

Reply:
One option is to expand it completely to "little file system" where it appears. Does that work for you?

Reply:
The terms `file`/`system`/`file system` are used somewhat commonly as well, referring to it as LittleFilesystem each time would make it seem more used IMO. I've been reading it for a while to be fair so I may be used to it, but I was less confused by the term than a bit inundated by how often it was used since there's not really a good pronoun to use for it to keep active voice.

Reply:
Well we're not describing what LittleFileSystem does, the class does very little. If you wanted to never mention LittleFS it sounds like that would have to wait for the technical docs mentioned below and link there?